### ISSUE TITLE
Unable to export to new GitHub repository

### ISSUE THREAD

**Issue by PoojMenon** [codesandbox/codesandbox-client#5926]
# Description
I'm trying to export to a new GitHub repository but keep getting thrown the below error: 
![image](https://user-images.githubusercontent.com/72031986/123265451-fb67f380-d4f2-11eb-8a5a-2f93331f7a2f.png)

## How has this issue affected you? What are you trying to accomplish?
I'm unable to commit my code to Github. 

### To Reproduce

![image](https://user-images.githubusercontent.com/72031986/123265698-3702bd80-d4f3-11eb-9832-2431a9f95dfc.png)

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      | Browser app              |
| Browser          | Chrome Version 91.0.4472.106           |
| Operating System | macOS BigSur Version 11.4              |

[code_of_conduct]: https://github.com/codesandbox/codesandbox-client/blob/master/CODE_OF_CONDUCT.md
[contributing]: https://github.com/codesandbox/codesandbox-client/blob/master/CONTRIBUTING.md

**Comment by KeitoKat** [comment]
Have the same issue here, but no solution is found yet :/

**Comment by Deep1Shikha** [comment]
I face the same issue, even I tried adding the newly created github link, but it does not seem to work.

**Comment by lbogdan** [comment]
Can you please look at the failed request in the devtools "Network" tab and paste (or screenshot) its body? Thanks!

**Comment by jasonfry** [comment]
I also have the same issue. I've tried in firefox and chrome, and can't import or export.

When trying https://githubbox.com/jasonfry/csbtest to automatically create a sandbox from Github it does a call to `csbtest?teamId=etcetc...` and gets a 422 with a body of `{"error":"UNKNOWN_ERROR - 401 - Bad credentials"}`

**Comment by JamesACS** [comment]
@jasonfry - We've recently made some changes to our Github flow - could you give this another shot for us?

**Comment by jasonfry** [comment]
@JamesACS - Hi. No I still get the same errors.

When trying to import:
```
We were unable to fork the sandbox
UNKNOWN_ERROR - 401 - Bad credentials
```

And when trying to export:
```
Unable to create the repo. Please refresh and try again or report issue.
UNKNOWN_ERROR - 401 - Bad credentials
```

**Comment by sannek** [comment]
@jasonfry I just had a look at this, and I think somehow we had an invalid Github token for you in our database. I've manually deleted that for now. Could you reconnect your github account (via the sidebar in any sandbox) and try again?

How this happened is a different story that I'll need to figure out, but hopefully that will at least fix it!

**Comment by jasonfry** [comment]
@sannek yep, that now works, big thanks!

**Comment by sannek** [comment]
@jasonfry Excellent! Glad it works: one question to help me figure out what happened here: Any chance you were inactive on CodeSandbox for a long time and came back? (The most straighforward explanation is that the token somehow got revoked, not that it has always been invalid, so just wondering)

**Comment by jasonfry** [comment]
@sannek Yes I'm pretty sure that was the case.

**Comment by nicasi** [comment]
@sannek i face the same problem. Is it possible to update the revoked token myself?

**Comment by sannek** [comment]
Yes you can! Unfortunately there's no direct link to where you can do that, but here's how to get to the option:

Click the three dots in the top right to go to the menu, and then select `preferences`. In the modal that pops up, select `integrations` on the right, and click "sign out" / and then "sign in" again for Github. This will delete your previous token and store a new one when you sign in again.

Let me know if that still doesn't work - See screenshots below in case it's unclear.

![image](https://user-images.githubusercontent.com/17491062/147267524-1b1113c4-557d-44b2-a204-08b92bb90178.png)
![image](https://user-images.githubusercontent.com/17491062/147267673-3d2c00e5-0320-438a-bbf1-c24cd5a681b1.png)

**Comment by nicasi** [comment]
> Yes you can! Unfortunately there's no direct link to where you can do that, but here's how to get to the option:
> 
> Click the three dots in the top right to go to the menu, and then select `preferences`. In the modal that pops up, select `integrations` on the left, and click the red X to sign out / and then sign in again for Github. This will delete your previous token and store a new one when you sign in again.

That fixed the issue for me. Thanks!
